app/assets/stylesheets/dance.scss

```diff
@@ -0,0 +1,3 @@
+#dance_comment_list li {
```

Comment:
Just to expand on this topic a little more, typically classes and ids nowadays are not used for their intended purpose.

classes are for targeting with CSS, ids are used for targeting with JS, although ids can be used sparingly to style unique elements on a page in which using a class would not work